src/index.html

```diff
@@ -7,8 +7,70 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
```

Comment:
you should use `a` tag as a container and control height and width of the image

Reply:
i did